CHANGELOG.md

```diff
@@ -21,6 +21,7 @@
 
 ## Bug Fixes
 
+- Added `CMakeLists.txt` to `Manifest.in` ([#4101](https://github.com/pybamm-team/PyBaMM/pull/4101))
```

Comment:
```suggestion
```

I would suggest removing this for now. We can add a CHANGELOG entry later for changes that are more user-facing (I don't think anyone is attempting to build PyBaMM from source from the source distribution right now and the IDAKLU solver still can't be compiled unless we include `pybind11`, so cloning the GitHub repository is better in this case).